src/data/content/design-system/fondamenti/griglia.yaml

```diff
@@ -0,0 +1,273 @@
+# ————————————————————————————————————————————
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  json: bsi/organizzare-gli-spazi/griglie.json
+  template:
+    name: design-system-index
+
+seo:
+  name: Griglia
+  description: "Nella definizione dell'interfaccia utente la griglia è la suddivisione dello spazio che scandisce il ritmo dei contenuti"
```

Comment:
```suggestion
  description: "La griglia di impaginazione rappresenta la struttura invisibile che permette di organizzare i contenuti della pagina"
```

Reply:
@zetareticoli @Fupete userei forse la stessa def. presente nel manuale operativo

Reply:
Sì, possiamo usarne contenuti anche per altri di questi fondamenti operativi. Qui il paragrafo specifico da cui attingere https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/it/versione-corrente/doc/esperienza-utente/progettare-e-costruire-in-alta-fedelta.html#le-griglie

Ricordiamoci che dopo la versione beta dovremo aggiornare anche tutto il capitolo 6 del Manuale creando riferimenti trasversali col Design system.